docs/examples/testcard/index.html

```diff
@@ -708,3 +708,38 @@ <h6>Box style 4 with <a href="go.warwick.ac.uk/its">link heading 6</a> </h6>
     {% include config-options.html %}
   </aside>
 </div>
+
+<hr />
+<h2>Utility classes for margin on floated items</h2>
```

Comment:
The proper place to document stuff is on components/main-content rather than on the test card